### ISSUE TITLE
ForeignKey missing 1 required positional argument  `references`

### ISSUE THREAD

**Issue by sumitsharansatsangi** [piccolo-orm/piccolo#745]
I have written following, but receives following error.
`region = ForeignKey(references=Region, on_delete=OnDelete.no_action, on_update=OnUpdate.no_action)`

Please check it above line is true or not.
I am receiving error on in this line.
The error is as follows:-
```
 region = ForeignKey(references=Region, on_delete=OnDelete.no_action, on_update=OnUpdate.no_action)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/user/PycharmProjects/asphalt/venv/lib/python3.11/site-packages/piccolo/columns/column_types.py", line 1871, in __init__
    references._meta.primary_key.__class__(
TypeError: ForeignKey.__init__() missing 1 required positional argument: 'references'

```

**Comment by dantownsend** [comment]
OK, interesting - what's the primary key on the `Region` table. Have you specified a custom one?

**Comment by sumitsharansatsangi** [comment]
`Region` Table has multiple primary key. I think this is creating the issue. How can I write composite primary key ?

**Comment by sumitsharansatsangi** [comment]
Ok, is there a way for composite uniqueness of two or more column?
If not , Will manually adding constraint using sql will add any problem ?

**Comment by dantownsend** [comment]
Yeah, adding a composite constraint using SQL is probably your best option.

**Comment by sumitsharansatsangi** [comment]
Thanks, I am going with SQL.
But I request you , if you have time,  please check SQLAlchemy implementation . SQLAlchemy supports both, composite primary key and composite uniqueness. I think it can be done.

**Comment by dantownsend** [comment]
You're right - it can be done. Composite unique keys is something I want to get done soon.

**Comment by sumitsharansatsangi** [comment]
Good to know. Thanks.
